runtime/near-vm-runner/src/wasmer1_runner.rs

```diff
@@ -243,11 +245,31 @@ pub(crate) fn compile_wasmer1_module(code: &[u8]) -> bool {
     Module::new(&store, code).is_ok()
 }
 
+#[derive(Hash)]
+struct Wasmer1Config {
+    seed: i32,
+    use_cranelift: bool,
+    use_native_engine: bool,
+}
+
+impl Wasmer1Config {
+    fn config_hash(self: Self) -> u64 {
+        let mut s = DefaultHasher::new();
+        self.hash(&mut s);
+        s.finish()
+    }
+}
+
+const WASMER1_CONFIG: Wasmer1Config =
+    Wasmer1Config { seed: 53, use_cranelift: false, use_native_engine: false };
```

Comment:
If we are being fancy, we can also do

```rust
impl Wasmer1Config {
    const CURRENT: Self = Self { ... }
}
```

Seems a touch namespace-er, but doesn't really matter.